insights/parsers/tests/test_gfs2_file_system_block_size.py

```diff
@@ -0,0 +1,26 @@
+from insights.parsers import gfs2_file_system_block_size
+from insights.tests import context_wrap
+from insights.parsers import SkipException
+import pytest
+import doctest
+
+BLOCK_SIZE_OUTPUT = """
+4096
+""".strip()
+
+BLOCK_SIZE_OUTPUT_2 = """
+abc
+""".strip()
+
+
+def test_exp():
+    with pytest.raises(SkipException):
+        gfs2_file_system_block_size.GFS2FileSystemBlockSize(context_wrap(BLOCK_SIZE_OUTPUT_2))
+
+
```

Comment:
@huali027 you should add some additional testing for the parser here, and not rely solely on the documentation test and negative test.

Reply:
@bfahr, I thought it was enough to test right value and bad value. Since the output is every simple. Let me check to add more test, thanks for the review.